_posts/blog/2018-10-10/2018-10-10-how-to-buy-cryptocurrencies-privately-with-no-fees.markdown

```diff
@@ -0,0 +1,86 @@
+---
+title: "How to Buy Cryptocurrencies <span class='post-title-extra'>Privately with no Fees</span>"
+layout: post
+date: '2018-10-10 06:30:00 -0300'
+social_title: 'How to Buy Cryptocurrencies Privately with no Fees'
+social_description: 'This tutorial explains how to use OpenBazaar to purchase cryptocurrencies.'
+---
+
+This tutorial will explain how to purchase cryptocurrencies on OpenBazaar, a decentralized marketplace which never has fees and doesn’t require opening an account. This is for people buying cryptocurrency with a different cryptocurrency, such as Bitcoin. At the moment OpenBazaar doesn’t support people buying cryptocurrency with regular fiat money such as US Dollars or Euros, though it’s a feature that will hopefully be supported in the future.
+
+Here’s a video tutorial explaining the process:
+
+{% include modules/embeded-video.html url="https://www.youtube.com/embed/q6ki3_1Io-U"%}
+
+
+
+## Step #1: Download OpenBazaar
+
+Visit [OpenBazaar.org](https://openbazaar.org/download) and click on the installer for your operating system. Once downloaded, open the file and it will begin the process of installing OpenBazaar.
```

Comment:
I would say "installation link", since they're not actually clicking on the installer itself (pedantic, I know, but it could confuse some people).

Reply:
> Once downloaded

"Once the installer file is downloaded"